default-views/account/register-form.hbs

```diff
@@ -9,7 +9,30 @@
             <label class="control-label" for="username">Username*</label>
             <input type="text" class="form-control" name="username" id="username" placeholder="alice"
                    required/>
-          </div>
+
+	    {{#if multiuser}}
+	    <p>Your username should be a lower-case word with only
+	    letters a-z and numbers 0-9 and without periods.</p>
+	    <p>Your public Solid POD URL will be:
+	    <tt>https://<span class="editable-username">alice</span>.<script type="text/javascript">
+              document.write(window.location.host)
+	    </script></tt></p>
```

Comment:
We could probably do this with server-side templating (passing in the hostname from `renderForm` in `create-account-request.js` if not already passed in). But this works too.

Reply:
Yeah, I looked into that, but it seems we'd have to hook in the `SolidHost` class to do it, and I figured it is not worth it.